tox.ini

```diff
@@ -1,28 +1,32 @@
 [tox]
-envlist = py3{5,6}-{debug,release}
+envlist = py3{5,6,7}-{debug,release}
 skip_missing_interpreters = True
-skipsdist = True
 
 [testenv]
 deps =
     -r{toxinidir}/requirements.txt
+commands = pytest tests {posargs}
 
-basepython:
-    py35: python3.5
-    py36: python3.6
+[testenv:flake8]
```

Comment:
Now, include lint envs as separate jobs in Travis CI.

Reply:
Not sure whether I have permissions to do this in Travis CI

Reply:
It's about editing `.travis.yml`.

Reply:
oh, I see, added.